lib/jobs.rb

```diff
@@ -0,0 +1,12 @@
+require 'json'
+
+class Jobs
+  def restock
+    items = Item.all
+    restock_debug = 'Need to restock these: '
```

Comment:
Why don't we just return the items and the locations where there are low quantities of that item as a string, rather than a item json as a string?

That will make it easier when the `groot-notification-service` is up to send this notification, as the message will contain only the useful information.

Reply:
I was just using the string for debugging right now because when the notification service is ready I was thinking of just sending the request from this function.

Reply:
Yeah that's fine, just curious why we would want `i.serialize.to_json` rather than just `i.name`, or `"#{i.name (min(i.vending.map(:quantity))}"` or something like that (something more readable I mean)